src/Common/tests/DefaultOptionsProviderTests.cs

```diff
@@ -0,0 +1,73 @@
+﻿// ***********************************************************************
```

Comment:
Bracket the whole test class with #if !SILVERLIGHT && !NETCF - we like to do it that way and add the file everywhere because it facilitates searching for #ifs.